src/odemis/util/test.py

```diff
@@ -136,7 +136,15 @@ def stop_backend():
     model._core._microscope = None  # force reset of the microscope for next connection
 
     if status != driver.BACKEND_STOPPED:
-        raise IOError("Backend failed to stop, now %s" % status)
+        logging.warning("Backend failed to stop, will run odemis-stop to kill it.")
+        try:
+            ret = subprocess.run(["sudo", "odemis-stop"], timeout=60)
```

Comment:
I think there should be a TODO that this is a temporary solution for stopping the back-end but that a better solution should be found one day.